app/assets/stylesheets/style.scss

```diff
@@ -1 +1,13 @@
+body {
+   display: flex;
+   min-height: 100vh;
+   flex-direction: column;
+ }
 
+ main {
+   flex: 1 0 auto;
+ }
+
+.flash_notice {
+  background-color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.